torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -235,9 +235,10 @@ void ConvertGraphToONNXProto(
     std::shared_ptr<Graph> graph,
     onnx::ModelProto& model_proto,
```

Comment:
Can we update argument to `std::shared_ptr<onnx::ModelProto>& model_proto`?

Reply:
this argument is used in subsequent functions in InferShapes, present in implementation.cc/h. This would require importing ModelProto into that file as well

Reply:
You can still pass in by value. Besides, implementation.cc is part of onnx, so it has ModelProto imported already.